### DISCUSSION TITLE
Failed in Compiling MET-11.1.0

### DISCUSSION THREAD

**Discussion by DanielMwageni** [dtcenter/METplus#2504] [Configuration] [answered]
I have tried several times to compile MET in my Ubuntu laptop using the instructions from the dtcenter website but failes. The libraries compile successfully without any errors but when MET starts to compile it returns the following error ERROR: Command returned with non-zero (2) status: make -j. I have read previous discussions and tried various solutions but it does not work. Below are my environment config file and the met.make.log file
[install_met_env2.txt](https://github.com/dtcenter/METplus/files/14354542/install_met_env2.txt)
[install_met_env3.txt](https://github.com/dtcenter/METplus/files/14354543/install_met_env3.txt)
[met.make.log](https://github.com/dtcenter/METplus/files/14354544/met.make.log)

**Comment by jprestop** [comment]
Hi @Dangerbringer. I'm sorry to hear about the trouble you are experiencing.  Could you please also pass along the `config.log` file in your MET-11.1.0 subdirectory?

**Comment by DanielMwageni** [comment]
Sure, here is my config.log file
[config.log](https://github.com/dtcenter/METplus/files/14367519/config.log)

**Comment by jprestop** [comment]
Thanks @Dangerbringer.  

Can you please tell me if the config.log and met.make.log file that you provided corresponds with your run using [install_met_env2.txt](https://github.com/dtcenter/METplus/files/14354542/install_met_env2.txt) or [install_met_env3.txt](https://github.com/dtcenter/METplus/files/14354543/install_met_env3.txt)?

**Comment by jprestop** [comment, marked as answer]
And @Dangerbringer, I have suggestion for you to try.  In your /home/daniel/miniconda3/ environment, please install the libstdcxx-ng package (`conda install -c conda-forge libstdcxx-ng`) and then try recompiling MET.  Looking at the compiler being used in the config.log file ("gcc (Ubuntu 13.2.0-4ubuntu3) 13.2.0"), it looks like you should be using the install_met_env2.txt file, but please try changing: `export MAKE_ARGS="-j"` to `export MAKE_ARGS="-j 5"` as we have seen some trouble with not setting a value.

**Comment by DanielMwageni** [comment]
Thanks a lot @jprestop, after installing the libstdcxx-ng package and changing `MAKE_ARGS="-j"` to `export MAKE_ARGS="-j 5`", MET was compiled successfully. Thanks

**Reply by jprestop** [reply]
@Dangerbringer Great news!  Thank you so much for letting us know.  I'll lock this discussion to prevent future posts here.  Please don't hesitate to create a new discussion with any new problems or questions.
